docs/getting_started/typescript.md

```diff
@@ -153,7 +153,8 @@ Following are the currently allowed settings and their default values in Deno:
     "disableSizeLimit": false,
     "generateCpuProfile": "profile.cpuprofile",
     "jsx": "react",
-    "jsxFactory": "React.createElement",
+    "jsxFactory": "h",
```

Comment:
This is incorrect.  Why do you think this is the default?

Reply:
I updated the values from https://github.com/denoland/deno/blob/98c9798cb757045b3f46573c5baa80e2cfe672aa/cli/ast.rs#L175-L191

Is that correct now?